segmentation/utils/spatialanalysis_utils_test.py

```diff
@@ -10,3 +14,138 @@ def test_calc_dist_matrix():
     dist_matrix = spatialanalysis_utils.calc_dist_matrix(test_mat)
     real_mat = np.array([[0, 5], [5, 0]])
     assert np.array_equal(dist_matrix, real_mat)
+
+
+"""
+def test_distmat():
+    testtiff = io.imread(
+        "/Users/jaiveersingh/Documents/MATLAB/SpatialAnalysis/newLmod.tiff")
+    distmat = np.asarray(pd.read_csv(
+        "/Users/jaiveersingh/Documents/MATLAB/distancesMat5.csv", header=None))
```

Comment:
triple quotes are only used for docstrings: comment these out with # comment character instead. 
You can comment/uncomment multiple lines by highlighting then pressing cmd + /